requirements.txt

```diff
@@ -15,4 +15,4 @@ tensorboard
 # pycocotools corresponds to https://github.com/ppwwyyxx/cocoapi
 pycocotools>=2.0.2
 onnx>=1.13.0
-onnx-simplifier==0.4.10
```

Comment:
@kwadkore 
Could you please tell me the reason behind this change?
This is how I got by, but I'd like to determine what would be the appropriate solution.
https://github.com/Megvii-BaseDetection/YOLOX/issues/1801